README.md

```diff
@@ -24,6 +24,11 @@ $ npm i
 $ npm start
 ```
 
+## Running Tests
+
+1. `./node_modules/.bin/mocha test/**/*.js` to run all tests.
```

Comment:
The `mocha` script in `package.json` should also be modified to run this. (And likewise the `test` script should be changed to `npm run lint && npm run mocha`.)